symplyphysics/laws/astronomy/maximum_angle_of_rotation_during_gravitational_maneuver.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve, atan)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+
+# Description
+## A gravitational maneuver is a purposeful change in the trajectory and flight speed of a spacecraft under the influence
+## of the gravitational fields of celestial bodies.
+## The maximum angle of rotation of the rocket during a gravitational maneuver depends on the first cosmic velocity of
+## the planet around which the maneuver is performed, and the speed of the rocket relative to this planet.
+
+## Law is: phi = arctg((v1 / v2)^2), where
```

Comment:
Btw, do you have a link to this law?

Reply:
> Btw, do you have a link to this law?

https://college.ru/astronomy/course/content/chapter3/section2/paragraph3/theory.html
![image](https://github.com/blackyblack/symplyphysics/assets/100017508/d6c879ca-a207-48c3-815d-24cd4b3e8f42)